### PR TITLE
[WIP] refactor(organisms): SfList

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
https://github.com/DivanteLtd/storefront-ui/issues/806

# Scope of work
refactor SfList

# Screenshots of visual designs
mobile:
image

desktop:
image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))
 - [ ] I accept [Individual Contributor License Agreement](https://github.com/DivanteLtd/next/blob/master/CLA.md)

### CODE REVIEW

File: packages/shared/styles/components/organisms/SfList.scss [hunk 1]
```diff
@@ -1,22 +1,22 @@
 @import "../../helpers";
 
 .sf-list {
-  list-style-type: var(--list-list-style-type, none);
+  list-style: var(--list-list-style, none);
   padding: var(--list-padding, 0);
```
Comment (pspaczek) [inline packages/shared/styles/components/organisms/SfList.scss]:
```suggestion
  padding: var(--list-padding, 0);
  margin: var(--list-margin, 0);
```

File: packages/shared/styles/components/organisms/SfList.scss [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+    margin: var(--list-item-margin, 0);
+    color: var(--list-item-color, var(--c-text));
+    @include font(
+      --list-item-font,
+      var(--font-medium),
+      var(--font-base),
+      1.9,
+      var(--font-family-secondary)
     );
+    letter-spacing: 0.1px;
```
Comment (pspaczek) [inline packages/shared/styles/components/organisms/SfList.scss]:
```suggestion
```

File: packages/shared/styles/components/organisms/SfList.scss [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
-        1.6,
-        var(--font-family-primary)
-      )
+    margin: var(--list-item-margin, 0);
+    color: var(--list-item-color, var(--c-text));
+    @include font(
+      --list-item-font,
+      var(--font-medium),
+      var(--font-base),
+      1.9,
```
Comment (pspaczek) [inline packages/shared/styles/components/organisms/SfList.scss]:
```suggestion
      var(--font-regular),
      var(--font-sm),
      1.6,
```

File: packages/shared/styles/components/organisms/SfList.scss [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
 
 .sf-list {
-  list-style-type: var(--list-list-style-type, none);
+  list-style: var(--list-list-style, none);
   padding: var(--list-padding, 0);
-  margin: var(--list-margin, 0);
   @include for-desktop {
-    --list-font-size: var(--font-size-small);
+    --list-item-font-line-height: 2.4;
+    --list-item-font-weight: var(--font-light);
```
Comment (pspaczek) [inline packages/shared/styles/components/organisms/SfList.scss]:
```suggestion
```

File: packages/vue/src/components/organisms/SfList/SfList.stories.js [hunk 5]
```diff
@@ -26,7 +26,7 @@ storiesOf("Organisms|List", module).add("Common", () => ({
         :key="item.label" 
         :style="{margin: '10px 0'}"
       >
-        <SfMenuItem :label="item.label" :count="item.count"/>
+        <div>{{item.label}}</div>
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfList/SfList.stories.js]:
```suggestion
        <SfMenuItem :label="item.label" :count="item.count"/>
```
in example you should present commo user scenerio, when user copy story, then it should have right with SFUI elements

Comment (AdamPawlinski) [reply packages/vue/src/components/organisms/SfList/SfList.stories.js]: ok thanks for explanation.

**Pull request merged**